### PR TITLE
update :repo of many github packages

### PR DESCRIPTION
**Pull request by tarsius**
Before making these changes I did double check using the Github Api.

### CODE REVIEW

File: recipes/editorconfig-core [hunk 1]
```diff
@@ -1,3 +1,3 @@
-(editorconfig-core :repo "editorconfig/editorconfig-emacs"
+(editorconfig-core :repo "10sr/editorconfig-core-emacslisp"
```
Comment (10sr) [inline recipes/editorconfig-core]: I moved `editorconfig-core` to editorconfig/editorconfig-emacs so no need to change this :innocent: 
`10sr/editorconfig-core-emacslisp` is not maintained any more.

File: recipes/editorconfig-fnmatch [hunk 2]
```diff
@@ -1,3 +1,3 @@
-(editorconfig-fnmatch :repo "editorconfig/editorconfig-emacs"
+(editorconfig-fnmatch :repo "10sr/editorconfig-fnmatch-el"
```
Comment (10sr) [inline recipes/editorconfig-fnmatch]: Same as :point_up:

File: recipes/editorconfig-core (hunk 1 above)
Comment (tarsius) [inline recipes/editorconfig-core]: Thanks. I have reverted these changes and will fix this on the Emacsmirror instead.

**Comment by tarsius** [comment]
I have now reviewed all cases where both the repository tracked by the Emacsmirror and the one tracked by Melpa still exist. In most cases it actually was Melpa which was correct. I have dropped the changes which "fixed" the respective recipes and will instead fix the Emacsmirror.

**Comment by tarsius** [comment]
The following packages are still available from both places and I think the one tracked by the Emacsmirror is the correct one:

* `google` by @hober was forked by @steckerhalter and @hober then merged the additional commit.
* `plantuml-mode` by @zwz was forked by @wildsould who then made some changes in 2013. In 2014 and 2015 @zwz made changes too. History diverged.
* `ponylang-mode` @purcell's repo was forked by @SeanTAllen, which was then forked by @abingham 
* `qml-mode` by @xwl was forked by @coldnew. Both repos are at the same commit.

Undecided:

* `point-stack` by @mattharrison was forked by @dgutov. History diverged.
* `pomodoro` two independent implementations by @baudtack and @vderyagin. @baudtack's version was updated more recently.
* `ponylang-mode` @purcell's repo was forked by @SeanTAllen, which was then forked by @abingham. The latter two are at the same commit, the former lags behind 40 commits.

**Comment by SeanTAllen** [comment]
@tarsius I'm now the official maintainer of ponylang-mode. My repo should be considered the canonical one.

**Comment by tarsius** [comment]
Thanks @SeanTAllen!

**Comment by dgutov** [comment]
> point-stack by @mattharrison was forked by @dgutov. History diverge

 I'm really not sure which fork should be used. My fork is not just a cleanup or upgrade. It implements different, though arguably better, logic for storing the stack of positions.

**Comment by purcell** [comment]
Thanks for this @tarsius. Huge job to review it all though, sorry, will take me a while. So far I've noticed that `qml-mode` and `plantuml-mode` changes feel a little spurious and undesirable.

**Comment by purcell** [comment]
Applied, with the exception of the qml-mode change. Thanks!

**Comment by tarsius** [comment]
Thanks for the review!

The Emacsmirror now gets `qml-mode` from the same repository as Melpa.

**Pull request closed without merging**